book/src/prefabs/how_to_define_prefabs_aggregate.md

```diff
@@ -29,101 +24,91 @@ If you intend to include a [`Component`] that has not yet got a corresponding [`
         Error,
     };
     use serde::{Deserialize, Serialize};
+    use specs_derive::Component;
     ```
 
 3. Define the aggregate prefab data type.
 
     In these examples, `Named`, `Position`, and `Weapon` all derive [`PrefabData`].
```

Comment:
This line and the code snippet don't match up any more; I think the enum prefab data should come under here now

Reply:
Hmm, actually I think I disagree. It shows an example of grouping multiple `PrefabData`s into an aggregate first and then moves to the more complex example of using an enum to support heterogeneous types. That seems like the right flow to me.

Reply:
oh yes, read the whole thing (instead of just the snippet) and it makes sense